unified_planning/test/test_grounder.py

```diff
@@ -82,18 +78,18 @@ def test_robot_locations_connected(self):
     def test_robot_locations_connected_from_factory(self):
         problem = self.problems['robot_locations_connected'].problem
 
-        with Grounder(name='up_grounder') as embedded_grounder:
-            self.assertTrue(embedded_grounder.supports(problem.kind))
-            ground_result = embedded_grounder.ground(problem)
-            grounded_problem, rewrite_plan_funct = ground_result.problem, ground_result.lift_action_instance
+        with Compiler(name='up_grounder') as grounder:
+            self.assertTrue(grounder.supports(problem.kind))
+            res = grounder.compile(problem, CompilationKind.GROUNDER)
```

Comment:
```suggestion
        with Compiler(name='embedded') as comp:
            self.assertTrue(grounder.supports(problem.kind))
            res = comp.compile(problem, CompilationKind.GROUNDER)
```
With the embedded compiler this will look better :smile: